metadata-ingestion/src/datahub/ingestion/source/sql/mssql/source.py

```diff
@@ -174,7 +174,10 @@ def __init__(self, config: SQLServerConfig, ctx: PipelineContext):
     @staticmethod
     def _add_output_converters(conn: Connection) -> None:
         def handle_sql_variant_as_string(value):
-            return value.decode("utf-16le")
+            try:
+                return value.decode("utf-16le")
+            except UnicodeDecodeError:
+                return value.decode("Windows-1251")
```

Comment:
This fix seems specific to the character encoding but looks okay to me as it doesn't change original success path.

Reply:
I faced errors with some servers `datahub.ingestion.run.pipeline.PipelineInitError: Failed to configure the source (mssql): 'utf-16-le' codec can't decode byte 0xe8 in position 34: truncated data` but another servers worked fine. This fix solves theese errors.